src/librustdoc/passes/mod.rs

```diff
@@ -348,3 +360,49 @@ impl fold::DocFolder for ImportStripper {
         }
     }
 }
+
+pub fn look_for_tests<'a, 'tcx: 'a, 'rcx: 'a, 'cstore: 'rcx>(
+    cx: &'a DocContext<'a, 'tcx, 'rcx, 'cstore>,
+    dox: &str,
+    item: &Item,
+    check_missing_code: bool,
```

Comment:
If we revisit this function, i'd like to replace this bool parameter with an enum that represents which check is occurring. It will make the callsite much easier to understand, IMO.